src/electron/device-connect-view/send-app-initialized-telemetry.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { TelemetryEventHandler } from '../../background/telemetry/telemetry-event-handler';
+import { TelemetryEventSource, TriggeredByNotApplicable } from '../../common/extension-telemetry-events';
+import { APP_INITIALIZED } from '../common/electron-telemetry-events';
+
+export const sendAppInitializedTelemetryEvent = (telemetryEventHandler: TelemetryEventHandler) => {
+    telemetryEventHandler.publishTelemetry(APP_INITIALIZED, {
+        telemetry: {
+            triggeredBy: TriggeredByNotApplicable,
+            source: TelemetryEventSource.ElectronAppInitialized,
```

Comment:
This should be `TelemetryEventSource.ElectronDeviceConnect` as the Device Connect is the general name for this view (the source doesn't point to the file itself... but to the view in general or components inside the UI)

Reply:
Updated. I got an impression that the variable source is suppose to kind of depict the purpose of the telemetry event; but I understand now that it is more about source of the event that is publishing the telemetry.